lib/active_shipping/carriers/fedex.rb

```diff
@@ -475,7 +475,8 @@ def parse_rate_response(origin, destination, packages, response, options)
 
       if success
         rate_estimates = xml.root.css('> RateReplyDetails').map do |rated_shipment|
-          service_code = rated_shipment.at('ServiceType').text
+          service_code = rated_shipment.at('ServiceType').try(:text)
+          raise ActiveShipping::ResponseError, "The response did not contain service type" unless service_code.present?
```

Comment:
Since it is a change in their API, I'm afraid this will just continue to the next attribute that isn't gonna be define neither. Do we have a copy of the response we're receiving? Could we run a test against it?